source/dyaml/exception.d

```diff
@@ -110,13 +111,16 @@ abstract class MarkedYAMLException : YAMLException
         @safe pure nothrow
     {
         super(problem ~ '\n' ~ problemMark.toString(), file, line);
+        mark = problemMark;
     }
 
     /// Construct a MarkedYAMLException from a struct storing constructor parameters.
     this(ref const(MarkedYAMLExceptionData) data) @safe pure nothrow
     {
         with(data) this(context, contextMark, problem, problemMark);
     }
+
+    Mark mark;
```

Comment:
Please move this to the beginning of the class.

Reply:
Done!
I also added a comment with `///` to generate a document for this member (e.g., the [document off ScannerException](https://dyaml.dpldocs.info/v0.8.4/dyaml.scanner.ScannerException.html)).